types/src/params.rs

```diff
@@ -141,6 +141,14 @@ impl<'a> Params<'a> {
 	pub fn into_owned(self) -> Params<'static> {
 		Params(self.0.map(|s| Cow::owned(s.into_owned())))
 	}
+
+	/// Return the byte length of the underlying data.
+	pub fn len(&self) -> usize {
```

Comment:
So, the docs on the type itself clear states that it's a wrapper of a `JSON string` thus it should okay to use
`len` here i.e, not to be confused with the length of params themselves.

/cc @lexnv @jsdw

Reply:
Makes sense. Also I can definitely rename it to something like `data_len()` or `byte_len()`, if it is clearer.

Reply:
yeah, let's do `len_bytes()` to be on the safe side

Reply:
I don't mind whether it's `len()` or `byte_len()` given the comment. If it is `len()` though than iirc clippy will normally want you to also add an `is_empty()` function to be happy.

Reply:
Sounds good. Renamed to `len_bytes()` as suggested.